osquery/core/tests/watcher_tests.cpp

```diff
@@ -261,4 +267,41 @@ TEST_F(WatcherTests, test_watcherrunner_watcherhealth) {
   runner.isWatcherHealthy(*test_process, state);
   EXPECT_EQ(2U, state.sustained_latency);
 }
+
+TEST_F(WatcherTests, test_watcherrunner_unhealthy_delay) {
+  FakeWatcherRunner runner(0, nullptr, true);
+
+  auto test_process = PlatformProcess::getCurrentProcess();
+  auto fake_test_process = FakePlatformProcess(test_process->nativeHandle());
+  fake_test_process.setStatus(PROCESS_STILL_ALIVE, 0);
+
+  // Set up a fake test process and place it into an unhealthy state.
```

Comment:
Do you mean `healthy state` here? Seems like the test transitions from `healthy` to `unhealthy`.